nautobot_circuit_maintenance/handle_notifications/handler.py

```diff
@@ -23,19 +24,19 @@
 
 
 def create_circuit_maintenance(
-    logger: Job, maintenance_id: str, parsed_notification: ParsedNotification, provider: Provider
+    logger: Job, maintenance_id: str, parser_maintenance, provider: Provider
 ) -> CircuitMaintenance:
     """Handles the creation of a new circuit maintenance."""
     circuit_maintenance_entry = CircuitMaintenance.objects.create(
```

Comment:
I just realized in testing my extension plugin that `CircuitMaintenance.objects.create()` doesn't trigger the `pre_save` or `post_save` signals, since it doesn't actually call `save()`. 😞 Would you be willing to change this to something like:

```python
circuit_maintenance_entry = CircuitMaintenance(
   ...
)
circuit_maintenance_entry.save()
```

so that the save signals are properly triggered as I need them to be?

Reply:
sure! it's the best timing 😄

Reply:
Thanks!